backbone/README.md

```diff
@@ -159,6 +160,35 @@ return MyItemView;
 
 **[⬆ back to top](#table-of-contents)**
 
+## Ordering
+
+  - **Outside** `Marionette.View` definition
+
+  0. `dependencies` requirement
+  0. `static` methods
+  0. `config` objects
+
+  - Ordering for `Marionette.View`:
```

Comment:
you could say first life cycle methods as a clarification (so it doesn't need to be by extension and it can be by comprehension )